apps/search/scripts/migrations/migration-utils.ts

```diff
@@ -1,20 +1,18 @@
 /* eslint-disable turbo/no-undeclared-env-vars */
-
-import { SaleorCloudAPL } from "@saleor/app-sdk/APL";
+import { aplType,saleorApp } from "../../saleor-app";
 
 export const verifyRequiredEnvs = () => {
+  if (aplType == "file") return;
```

Comment:
do you need that? running migrations is something you explicitly run via script. So you rather know when do you want to execute it. I dont think we need to hardcode any conditions here

Reply:
not needed anymore, since I use `const saleorAPL = saleorApp.apl;` - verifying envs are already there (saleor-app.ts)